icu4c/source/data/brkitr/ko.txt

```diff
@@ -0,0 +1,242 @@
+﻿// © 2016 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+// Generated using tools/cldr/cldr-to-icu/build-icu-data.xml
```

Comment:
As it says, this file is actually supposed to be generated, from xml data in data/xml/brkitr/. Take a look at data/xml/brkitr/ja.xml and add something similar for ko.

Reply:
Thank you, Peter.   I added ko.xml . Should I delete ko.txt change, then?

Reply:
ooops.  perhaps, I still need to keep ko.txt (even though it's not generated by the tool). Or, should I build a CLDR tool and generate from the xml file? The result should be identical (modulo space/format, etc).

Reply:
> Please add unit tests.

I added several lines of test to rbbi/RBBITest/TestExtended . Is it sufficient?

> Should the currently-identical lists of particles ("extensions") be pushed into the root data file? Or not useful because they should eventually diverge?

It turned out that it's not necessary to duplicate the list. In the long run, I'd argue that the list should be moved to root, but that's for another day. 

I added a comment about Korean particle list for the future.

Reply:
> Thank you, Peter. I added ko.xml . Should I delete ko.txt change, then?

No need to delete the ko.txt change now; the change to ko.xml will just prevent your ko.txt change from being eliminated next time there is a CLDR-ICU integration.

Reply:
Got it. Thank you.